content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -115,24 +82,85 @@ The Istio control plane must be installed with virtual machine integration enabl
     $ istioctl install -f "${WORK_DIR}"/vmintegration.yaml
     {{< /text >}}
 
-## Create files to transfer to the virtual machine
+## Create Namespace and Setup policies for VM
+1. Create Namespace that will host the VM
+
+    {{< text bash >}}
+    $ kubectl create namespace $SERVICE_NAMESPACE
+    {{< /text >}}
+
+1. Create a serviceaccount for VM
+
+    {{< text bash >}}
+    $kubectl create serviceaccount $SERVICE_ACCOUNT -n $SERVICE_NAMESPACE 
+    {{< /text >}}
+
+1. Get the external ip or internal ip of the VM and store to `"${VM_IP}"`
+
+    {{< text bash >}}
+    $ VM_IP="<the external ip or internal ip of the VM>"
+    {{< /text >}}
+    
+1. Create a new service
+    {{< text bash >}}
+    $ cat <<EOF | kubectl -n $SERVICE_NAMESPACE apply -f -
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: gce-vm
+      labels:
+        app: gce-vm
+    spec:
+      ports:
+      - port: 7070
+        name: grpc
+      - port: 8090
+        name: http
+      selector:
+        app: gce-vm
+    EOF
+    {{< /text >}}
+1. Create workload entry
+
+{{< text bash >}}
+$ cat <<EOF | kubectl -n $namespace apply -f -
+  apiVersion: networking.istio.io/v1beta1
+  kind: WorkloadEntry
+  metadata:
+    name: $VM_NAME
+    namespace: $SERVICE_NAMESPACE
+  spec:
+    address: $VM_IP
+    labels:
+      app: gce-vm
+    serviceAccount: $SERVICE_ACCOUNT
+  EOF
+{{< /text >}}
 
-1. Make a copy of files to copy to the virtual machine
+## Create files to transfer to the virtual machine
```

Comment:
meta comment: all of these steps could use explanation. We don't really want users copy+pasting random security sensitive stuff. But can be done as a followup